### PR TITLE
Clickhouse data

### PR DESCRIPTION
**Pull request by valoumiaou**
# DE PR

## Describe your changes

Please include a summary of the changes:

This PR imports data in clickhouse cluster

Tag a reviewer if necessacy  @github/username 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [x] New table (non-breaking change)
- [x] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] My code passes CI/CD tests
- [ ] I updated README.md
- [x] I have updated the dag
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [x] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?
- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] ⏰ no, but I created a ticket

### CODE REVIEW

File: orchestration/dags/data_gcp_dbt/models/export/clickhouse/exp_clickhouse__pro_events.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    event_timestamp,
+    offer_id as offer_id,
+    user_id as user_id,
+    unique_session_id,
+    origin
+FROM {{ source("analytics","firebase_pro_events") }}
+WHERE 
+    event_date BETWEEN date_sub(DATE("{{ ds() }}"), INTERVAL {{ var("clickhouse_day_params",4) }} DAY) and DATE("{{ ds() }}")
+AND user_pseudo_id is NOT NULL 
+-- AND event_name in ("ConsultOffer","ConsultVenue","HasAddedOfferToFavorites")
```
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/export/clickhouse/exp_clickhouse__pro_events.sql:13]:
```suggestion
```

File: orchestration/dags/data_gcp_dbt/models/export/clickhouse/exp_clickhouse__bookings.sql
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/export/clickhouse/exp_clickhouse__bookings.sql:1]: Pour les noms des modèles on met tout au singulier. :)

File: orchestration/dags/dependencies/export_clickhouse/export_clickhouse.py [hunk 2]
```diff
@@ -18,3 +18,24 @@
         "mode": "overwrite",
     },
 ]
+
+TABLES_CONFIGS2 = [
```
Comment (cdelabre) [inline orchestration/dags/dependencies/export_clickhouse/export_clickhouse.py:22]: replace with old config file

File: orchestration/dags/jobs/export/export_clickhouse2.py [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+from common.utils import get_airflow_schedule
+from common.alerts import task_fail_slack_alert
+from dependencies.export_clickhouse.export_clickhouse import TABLES_CONFIGS2
+from common.alerts import task_fail_slack_alert
+import copy
+from common import macros
+import os
+
+# SQL_BASE_PATH = Path(f"{os.environ['GCS_BUCKET']}/plugins/compiled/data_gcp_dbt/models/export/clickhouse")
```
Comment (cdelabre) [inline orchestration/dags/jobs/export/export_clickhouse2.py:33]: remove

File: orchestration/dags/jobs/export/export_clickhouse2.py [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+}
+DATE = "{{ yyyymmdd(ds) }}"
+
+dag_config = {
+    "STORAGE_PATH": f"{DATA_GCS_BUCKET_NAME}/clickhouse_export/{ENV_SHORT_NAME}/export/{DATE}",
+    "BASE_DIR": "data-gcp/jobs/etl_jobs/internal/export_clickhouse/",
+}
+
+gce_params = {
+    "instance_name": f"export-clickhouse2-data-{ENV_SHORT_NAME}",
```
Comment (cdelabre) [inline orchestration/dags/jobs/export/export_clickhouse2.py:57]: 2?

File: orchestration/dags/jobs/export/export_clickhouse2.py [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+        },
+    },
+}
+
+
+for dag_name, dag_params in dags.items():
+    with DAG(
+        f"export_clickhouse2_{dag_name}",
+        default_args=dag_params["default_dag_args"],
+        description="Export to analytics data posthog",
```
Comment (cdelabre) [inline orchestration/dags/jobs/export/export_clickhouse2.py:79]: posthog ?

File: orchestration/dags/data_gcp_dbt/models/export/clickhouse/exp_clickhouse__native_event.sql [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    venue_id,
+    origin,
+    is_consult_offer,
+    is_consult_venue,
+    is_add_to_favorites
+FROM {{ ref("int_firebase__native_event") }}
+WHERE 
+    event_date BETWEEN date_sub(DATE("{{ ds() }}"), INTERVAL {{ var("clickhouse_day_params",4) }} DAY) and DATE("{{ ds() }}")
+AND user_pseudo_id is NOT NULL 
+AND MOD(ABS(FARM_FINGERPRINT(user_pseudo_id)),10) = 0
```
Comment (cdelabre) [inline orchestration/dags/data_gcp_dbt/models/export/clickhouse/exp_clickhouse__native_event.sql:17]: ?

File: orchestration/dags/data_gcp_dbt/models/export/clickhouse/exp_clickhouse__native_event.sql [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    user_id as user_id,
+    unique_session_id,
+    venue_id,
+    origin,
+    is_consult_offer,
+    is_consult_venue,
+    is_add_to_favorites
+FROM {{ ref("int_firebase__native_event") }}
+WHERE 
+    event_date BETWEEN date_sub(DATE("{{ ds() }}"), INTERVAL {{ var("clickhouse_day_params",4) }} DAY) and DATE("{{ ds() }}")
```
Comment (cdelabre) [inline orchestration/dags/data_gcp_dbt/models/export/clickhouse/exp_clickhouse__native_event.sql:15]: Quelle utilité sur une vue de forcer 4 jours?

File: orchestration/dags/jobs/export/export_clickhouse2.py [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+            "instance_type": Param(
+                default=gce_params["instance_type"],
+                type="string",
+            ),
+            "instance_name": Param(
+                default=gce_params["instance_name"],
+                type="string",
+            ),
+            "days": Param(
+                default=-30,
```
Comment (cdelabre) [inline orchestration/dags/jobs/export/export_clickhouse2.py:103]: 30 ?

File: jobs/etl_jobs/internal/export_clickhouse/schema/analytics/daily_aggregated_event.sql [hunk 9]
```diff
@@ -0,0 +1,17 @@
+CREATE VIEW analytics.daily_aggregated_event ON cluster default
+AS
+SELECT
+    toDate(partition_date) AS event_date,
+    origin,
+    offer_id,
+    venue_id,
+    sum(is_consult_offer) AS nbr_offer_consultation,
```
Comment (cdelabre) [inline jobs/etl_jobs/internal/export_clickhouse/schema/analytics/daily_aggregated_event.sql:8]: nbr?

**Review by cdelabre — CHANGES_REQUESTED** [review]

File: jobs/etl_jobs/internal/export_clickhouse/schema/analytics/daily_aggregated_event.sql (hunk 9 above)
Comment (carolinemestre-passculture) [reply jobs/etl_jobs/internal/export_clickhouse/schema/analytics/daily_aggregated_event.sql:8]: `total_consultated_offers` ?

File: jobs/etl_jobs/internal/export_clickhouse/refresh.py
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/refresh.py:1]: @cdelabre

File: jobs/etl_jobs/internal/export_clickhouse/schema/analytics/daily_aggregated_event.sql
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/schema/analytics/daily_aggregated_event.sql:1]: @cdelabre

File: jobs/etl_jobs/internal/export_clickhouse/schema/analytics/monthy_aggregated_offerer_collective_revenue.sql
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/schema/analytics/monthy_aggregated_offerer_collective_revenue.sql:1]: @cdelabre

File: jobs/etl_jobs/internal/export_clickhouse/schema/analytics/monthy_aggregated_offerer_revenue.sql
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/schema/analytics/monthy_aggregated_offerer_revenue.sql:1]: @cdelabre

File: jobs/etl_jobs/internal/export_clickhouse/schema/analytics/yearly_aggregated_offerer_collective_revenue.sql
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/schema/analytics/yearly_aggregated_offerer_collective_revenue.sql:1]: @cdelabre

File: jobs/etl_jobs/internal/export_clickhouse/schema/intermediate/booking.sql
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/schema/intermediate/booking.sql:1]: @cdelabre

File: jobs/etl_jobs/internal/export_clickhouse/schema/intermediate/collective_booking.sql
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/schema/intermediate/collective_booking.sql:1]: @cdelabre

File: jobs/etl_jobs/internal/export_clickhouse/schema/intermediate/native_event.sql
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/schema/intermediate/native_event.sql:1]: @cdelabre

File: jobs/etl_jobs/internal/export_clickhouse/schema/tmp/intermediate_booking.sql
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/schema/tmp/intermediate_booking.sql:1]: @cdelabre

File: jobs/etl_jobs/internal/export_clickhouse/schema/tmp/intermediate_collective_booking.sql
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/schema/tmp/intermediate_collective_booking.sql:1]: @cdelabre

File: jobs/etl_jobs/internal/export_clickhouse/schema/tmp/intermediate_native_event.sql
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/schema/tmp/intermediate_native_event.sql:1]: @cdelabre

File: jobs/etl_jobs/internal/export_clickhouse/update.py
Comment (valoumiaou) [inline jobs/etl_jobs/internal/export_clickhouse/update.py:1]: @cdelabre

File: orchestration/dags/dependencies/export_clickhouse/export_clickhouse.py
Comment (valoumiaou) [inline orchestration/dags/dependencies/export_clickhouse/export_clickhouse.py:1]: @cdelabre

**Review by cdelabre — APPROVED** [review]

**Review by cdelabre — DISMISSED** [review]

**Pull request merged**